TRIAGING.md

```diff
@@ -0,0 +1,51 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
+The maintainers of the OpenSearch-Dashboards Repo seek to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project. To learn more about contributing to the OpenSearch-security Repo visit the [Contributing](./CONTRIBUTING.md) documentation.
```

Comment:
Security repo?

Reply:
Thanks for catching, I've removed the reference to security repo.